### PR TITLE
Correct CSI driver installation documentation for self-managed Kubern…

### PR DESCRIPTION
**Pull request by Josh-Tigera**
…etes AWS installations

## Description
Correction for self-managed Kubernetes installation on AWS documentation as it pertains to CSI driver installation.

## Related issues/PRs
Correction follow-on to [PR 6967](https://github.com/projectcalico/calico/pull/6967)

## Todos

- ~~[ ] Tests~~
- [x] Documentation
- ~~[ ] Release note~~

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: calico/getting-started/kubernetes/self-managed-public-cloud/aws.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ Kubernetes Operations (kops) is a cluster management tool that handles provision
 
 
 > **Note**: {{site.prodname}} makes use of the Kubernetes Container Storage Interface (CSI) to support various types of volumes. The necessary drivers required for CSI
-> to function correctly in EKS clusters will no longer be present by default in clusters running Kubernetes 1.23. Please see the following link to ensure your cluster
+> to function correctly in AWS clusters using EBS volumes will no longer be present by default in clusters running Kubernetes 1.23. Please see the following link to ensure your cluster
```
Comment (lwr20) [inline calico/getting-started/kubernetes/self-managed-public-cloud/aws.md:25]: I think this is going to depend on what installer you're using.  I don't think you can say that _all_ methods of installing kubernetes clusters 1.23+ on AWS will not have EBS CSI by default?

Comment (Josh-Tigera) [reply calico/getting-started/kubernetes/self-managed-public-cloud/aws.md:25]: True it will largely depend on what installer you're using. I'll reword it to indicate as much.

**Review by lwr20 — APPROVED** [review]
